pyhik/watchdog.py

```diff
@@ -33,4 +34,5 @@ def pet(self):
 
     def stop(self):
         """ Stops the watchdog timer. """
-        self._timer.cancel()
+        if self._timer is not None:
```

Comment:
Can you change this to:
`if self._timer:`

The explicit `is not None` shouldn't be necessary.

Reply:
Thanks. 
Added the none check because it had fails under HomeAssistant on this, probably because stop being called prior to start

Reply:
2020-02-22 12:00:17 WARNING (Thread-3) [pyhik.hikvision] None Connection Failed (count=1). Waiting 10s. Err: Connection unsucessful.
home-assistant   | 2020-02-22 12:00:17 INFO (MainThread) [homeassistant.components.automation] Initialized trigger Zigbee Join Enabled
home-assistant   | Exception in thread Thread-3:
home-assistant   | Traceback (most recent call last):
home-assistant   |   File "/usr/local/lib/python3.7/site-packages/pyhik/hikvision.py", line 474, in alert_stream
home-assistant   |     raise ValueError('Connection unsucessful.')
home-assistant   | ValueError: Connection unsucessful.
home-assistant   |
home-assistant   | During handling of the above exception, another exception occurred:
home-assistant   |
home-assistant   | Traceback (most recent call last):
home-assistant   |   File "/usr/local/lib/python3.7/threading.py", line 926, in _bootstrap_inner
home-assistant   |     self.run()
home-assistant   |   File "/usr/local/lib/python3.7/threading.py", line 870, in run
home-assistant   |     self._target(*self._args, **self._kwargs)
home-assistant   |   File "/usr/local/lib/python3.7/site-packages/pyhik/hikvision.py", line 532, in alert_stream
home-assistant   |     self.watchdog.stop()
home-assistant   |   File "/usr/local/lib/python3.7/site-packages/pyhik/watchdog.py", line 36, in stop
home-assistant   |     self._timer.cancel()
home-assistant   | AttributeError: 'Watchdog' object has no attribute '_timer'
home-assistant   |